stabilizer.cpp fails to compile with opencv < 3.2

```
In file included from stabilizer.cpp:21:0:
stabilizer.h:31:21: error: ‘videostab’ is not a namespace-name
 using namespace cv::videostab;
                     ^~~~~~~~~
```
cv::videostab namespace is defined in <opencv2/videostab.hpp>.  In opencv version 3.2+, this header is automatically included from <opencv2/opencv.hpp>.  However, prior to version 3.2, files must explicitly include <opencv2/videostab.hpp>

--- a/plugins/smart/dvs/libdvs/stabilizer.h
+++ b/plugins/smart/dvs/libdvs/stabilizer.h
@@ -24,6 +24,7 @@
 #include <vector>
 #include <opencv2/core.hpp>
 #include <opencv2/opencv.hpp>
+#include <opencv2/videostab.hpp>
 
 #include "libdvs.h"
 

